crates/pink/pink-extension/src/lib.rs

```diff
@@ -46,6 +48,7 @@ impl Topics for Message {
     }
 }
 
+#[cfg(feature = "std")]
```

Comment:
Looks like basically we are assuming `feature = "std"` means outside ink. Substrate also followed a similar convention in the past (std means on the native client side, and no_std means inside the wasm runtime). However it turned out a pain -- there are scenario where the code is running outside the wasm runtime in a no_std environment (like before we have std in pruntime) -- and then the convention become a burden.

If my understanding is correct, maybe it's wise to define a new feature to distinguish the code in/outside the ink execution environment.

Reply:
Added a feature 'runtime_utils'